src/parsing/index.ts

```diff
@@ -218,16 +222,21 @@ const getOnChainPayResponse = ({
         valid: false,
         address: path,
         errorMessage: "Invalid amount in payment destination",
+        memo,
       }
     }
 
-    // will throw if address is not valid
-    address.toOutputScript(path, networks[network === "mainnet" ? "bitcoin" : network])
+    const isTaprootAddress = path.toLowerCase().startsWith("bc1p")
+    // will throw if address is not valid, however bitcoinjs-lib does not currently support taproot addresses so we must bypass this check if the address uese taproot
```

Comment:
```suggestion
    // FIXME will throw if address is not valid, however bitcoinjs-lib does not currently support taproot addresses so we must bypass this check if the address uese taproot
```

I think we should have a FIXME on this, because it should be a temp fix. I guess code can be remove when bitcoinjs-lib has taproot enable